src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -15,7 +15,7 @@
 from azure.mgmt.web.models import DatabaseType, ConnectionStringType, BuiltInAuthenticationProvider, AzureStorageType
 
 from ._completers import get_hostname_completion_list
-from ._constants import FUNCTIONS_VERSIONS, FUNCTIONS_RUNTIME_STACKS_JSON_PATHS
+from ._constants import FUNCTIONS_VERSIONS, FUNCTIONS_RUNTIME_STACKS_JSON_PATHS, FUNCTIONS_RUNTIME_STACKS_API_KEYS
```

Comment:
Nit: Can you remove _Runtime_ ? Functions runtime generally refers to functions host.